src/metrics/logic.py

```diff
@@ -38,14 +39,39 @@ def notify(obj):
     else:
         events.notify(obj)
 
+'''
+def recently_updated_articles():
+    "all articles whose associated metrics/citations have been updated in the last hour"
+    since = utils.utcnow() - timedelta(hours=2)
+    recent_citations = Q(citation__datetime_record_updated__gte=since)
+    recent_views_downloads = Q(metric__datetime_record_updated__gte=since)
+    return models.Article.objects.filter(recent_citations | recent_views_downloads).distinct()
+'''
+
+def recently_updated_citations():
+    "all articles whose associated metrics/citations have been updated in the last hour"
```

Comment:
it would look like last 2 hours?
Given we run this every day, it could be more robust to send whatever has been updated in the last 24 hours? It would sometimes send more events than necessary however, so 2 hours seems like a good trade-off (updates must always finish withing 2 hours)